src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -587,7 +605,7 @@ def launch_fluent(
         Number of processors. The default is ``None``, in which case ``1``
         processor is used.  In job scheduler environments the total number of
         allocated cores is clamped to value of ``processor_count``.
-    journal_file_names : str, optional
+    journal_file_names : Union[None, str, list[str]], optional
```

Comment:
We already have "optional" listed, so listing `None` is redundant, and I believe this would work better for the docstring (for example see https://numpydoc.readthedocs.io/en/latest/format.html#parameters):
```suggestion
    journal_file_names : str or list of str, optional
```